src/librustc_data_structures/bitvec.rs

```diff
@@ -17,14 +17,15 @@ pub struct BitVector {
 }
 
 impl BitVector {
+    /// Create a new bitvector of at least num_bits length.
     pub fn new(num_bits: usize) -> BitVector {
         let num_words = u64s(num_bits);
         BitVector { data: vec![0; num_words] }
     }
 
     pub fn contains(&self, bit: usize) -> bool {
         let (word, mask) = word_mask(bit);
-        (self.data[word] & mask) != 0
+        (self.data.get(word).cloned().unwrap_or(0) & mask) != 0
```

Comment:
>  Do you use this behavior in your analysis?

I didn’t answer this: yes, I do. The dead code removing pass uses this behaviour.